share/picongpu/tests/Pusher/lib/python/test/Pusher/MainTest.py

```diff
@@ -0,0 +1,325 @@
+"""
+This file is part of the PIConGPU.
+
+Copyright 2023 PIConGPU contributors
+Authors: Hannes Wolf
+License: GPLv3+
+
+Test script to determine if a pusher works correctly. This is done by the
+comparison of the radii of a particle at constatnt speed in a homogenious
+magnetic field. If the relative change between two periods is greater than 
+epsilon = 1e-5, the test fails. Also the absolute phasechange during one turn
+is regarded. It should be smaller than 0.2 rad.
+"""
+
+import openpmd_api as io
+import numpy as np
+import sys
+
+
+def get_params(path):
+    """creates a series of the data found at path.
+    path should have the form data_%T to provide access to all iterations.
+
+    returns:
+        series
+        parameter list... list of parameters of the simulation (units etc.)
+        """
+
+    # load series
+    series = io.Series(path, io.Access.read_only)
+
+    if len(series.iterations) == 1:
```

Comment:
Could you throw an error instead, that would allow avoiding the use of the bool variable `longer_series` for later checks.

Reply:
done